concourse/scripts/pipecleaner.py

```diff
@@ -27,9 +27,13 @@
 import sys
 import os
 import getopt
+import tempfile
+import subprocess
 
 
 class Pipecleaner(object):
+    def __init__(self, shellcheck_enabled):
+        self.shellcheck_enabled = shellcheck_enabled
```

Comment:
I wonder if we should perhaps check if the shellcheck binary exists and enable it based on that rather than having a flag? Unsure.

Regardless, the container that is now running pipecleaner in travis (and elsewhere) now needs shellcheck installing.